pkg/kubelet/network/dns/dns_test.go

```diff
@@ -62,6 +62,23 @@ func newTestConfigurer(recorder *record.FakeRecorder, nodeRef *v1.ObjectReferenc
 		},
 		Status: arktosv1.NetworkStatus{
 			DNSServiceIP: dnsVIPOfTestNetwork,
+			Phase:        "Ready",
```

Comment:
"ready" -> arktosv1.NetworkReady

Reply:
it is fine to use literal in test code for ease of code reading.